astro/src/content/docs/release-notes/index.mdx

```diff
@@ -44,6 +44,104 @@ import { YouTube } from '@astro-community/astro-embed-youtube';
+  </Issue>
+* <Issue issue="2909">
+  The PHP client library is not handling libcurl errors gracefully, making it difficult to troubleshoot integration problems when using this library. See the [client library issue](https://github.com/FusionAuth/fusionauth-php-client/issues/28) for more details.
+  </Issue>
+* <Issue issue="2917">
+  When downloading login records from <Breadcrumb>System -> Login Records</Breadcrumb>, the exported file format contains a place for zip code, however the zip code values are not being populated in the export.
+  </Issue>
+* <Issue issue="2928">
+  The `POST /api/user/registration` call is documented as returning a refreshTokenId, but the id is not being returned on the response.
+  </Issue>
+* <Issue issue="2929">
+  When a license containing Breached Password Detection is missing a key, a NullPointerException is generated. This error should be handled more gracefully.
+  </Issue>
+* <Issue issue="2932">
+  When editing a user's password in the FusionAuth admin UI after a new hashing scheme is set on the tenant, the password is not re-hashed using the new scheme. The re-hashing occurs as expected on a login or when the user changes their own password.
+  </Issue>
+
+### Enhancements
+* <Issue issue="1596">
+  The `rate limit` error message was added to the default theme messages to make it more obvious that it is customizable. The `[RateLimitedException]` message key was previously supported, but not easily discoverable.
+  </Issue>
+* <Issue issue="1619">
+  Add an additional refresh token revocation policy to revoke a one-time use token on reuse. This policy helps protect against token theft by revoking the token if it were to be stolen and reused.
+  </Issue>
+* <Issue issue="2378">
+  FusionAuth can now accept encrypted SAML assertions when acting as a SAML service provider. Support for encrypted assertions when FusionAuth is the SAML identity provider was added in version `1.47.0 `.
+  </Issue>
+* <Issue issue="2537">
+  API keys can now be optionally set to expire at a given date/time. An expired key will remain in the system but will cause a `401` response to be returned if it is used in an API call. Expiration can be edited, so a key's expiration can be extended, and a key's expiration can be set to a past date/time to force it int an expired and unusable state.
+  </Issue>
+* <Issue issue="2742">
+  Additional parameters are now accepted on the hosted backend `/app/login` and `/app/registration` endpoints. This means you can pass things like `login_hint`, `idp_hint`, and analytics parameters that will be available on the respective OAuth hosted pages.
+  </Issue>
+* <Issue issue="2859">
+  The First-time Setup wizard was improved with more descriptive and consistent text around using a Community plan license.
```

Comment:
```suggestion
  The First-time Setup Wizard was improved with more descriptive and consistent text around using a Community plan license.
```

Reply:
"wizard" is not capitalized the other times it's used as `First-time Setup wizard`, so I am keeping with convention here.